e2e/tests/functional/plugins/notebook/notebook.e2e.spec.js

```diff
@@ -67,10 +68,33 @@ test.describe('Default Notebook', () => {
 
 test.describe('Notebook section tests', () => {
     //The following test cases are associated with Notebook Sections
-    test.fixme('New sections are automatically named Unnamed Section with Unnamed Page', async ({ page }) => {
-        //Create new notebook A
-        //Add section
-        //Verify new section and new page details
+    test.beforeEach(async ({ page }) => {
+        //Navigate to baseURL
+        await page.goto('./', { waitUntil: 'networkidle' });
+
+        // Create Notebook
+        await createDomainObjectWithDefaults(page, {
+            type: 'Notebook',
+            name: "Test Notebook"
+        });
+    });
+    test('Default sections are automatically named Unnamed Section with Unnamed Page', async ({ page }) => {
```

Comment:
These two tests could probably be combined into one. That would save us a tiny bit of runtime on the e2e tests.

You could even use `test.step()` to separate out the different test cases if you wanted to

Reply:
makes sense! I just combined them into a single test